### PR TITLE
EWL 3095 - Ribbon header cleanup

### PR DESCRIPTION
**Pull request by martensc**
Ticket: https://issues.ama-assn.org/browse/EWL-3095

To test:

- Go to http://localhost:3000/?p=organisms-ribbon and http://localhost:3000/?p=organisms-ribbon-auth and review the style changes outlined in the ticket.

### CODE REVIEW

File: styleguide/source/_patterns/01-molecules/05-navigation/07-ribbon-dropdown/_ribbon-dropdown.scss [hunk 1]
```diff
@@ -44,17 +47,24 @@
...
+  box-shadow: 3px 3px 2px $black-55;
   z-index: 100;
 
-  &.is-active {
-    display: block;
+  @include breakpoint(740px) {
+    box-shadow: none;
   }
+
+  &.is-active {}
```
Comment (aCyborg) [inline styleguide/source/_patterns/01-molecules/05-navigation/07-ribbon-dropdown/_ribbon-dropdown.scss]: Can we remove the is-active state here as it is no longer being used?

File: styleguide/source/_patterns/01-molecules/05-navigation/08-ribbon-user-menu/_ribbon-user-menu.scss [hunk 2]
```diff
@@ -36,18 +37,17 @@
   right: 0;
   z-index: 100;
   background-color: $black-20;
-  box-shadow: 3px 3px 2px $black-55;
+  box-shadow: -3px 3px 2px $black-55;
 
-  &.is-active {
-    display: block;
-  }
+  &.is-active {}
```
Comment (aCyborg) [inline styleguide/source/_patterns/01-molecules/05-navigation/08-ribbon-user-menu/_ribbon-user-menu.scss]: Remove this as well.

File: styleguide/source/_patterns/01-molecules/05-navigation/08-ribbon-user-menu/_ribbon-user-menu.scss [hunk 3]
```diff
@@ -36,18 +37,17 @@
...
 
   .ribbon_user-menu_nav_item.divider_item {
     display: none;
   }
 
   @include breakpoint(740px) {
-    display: block;
+    display: block !important;
+    height: auto !important;
```
Comment (aCyborg) [inline styleguide/source/_patterns/01-molecules/05-navigation/08-ribbon-user-menu/_ribbon-user-menu.scss]: Any workarounds to avoid !important?

**Comment by aCyborg** [comment]
@chelsiejohnston I reviewed this and for the most part it looks good to me, just a few cleanup changes. Would you like to review as well?

File: styleguide/source/_patterns/01-molecules/05-navigation/08-ribbon-user-menu/_ribbon-user-menu.scss (hunk 3 above)
Comment (martensc) [inline styleguide/source/_patterns/01-molecules/05-navigation/08-ribbon-user-menu/_ribbon-user-menu.scss]: @aCyborg I'd like to but the js `slideToggle()` prints out inline styles which takes precedence over file styles. I can take another look though.

File: styleguide/source/_patterns/01-molecules/05-navigation/08-ribbon-user-menu/_ribbon-user-menu.scss (hunk 2 above)
Comment (martensc) [inline styleguide/source/_patterns/01-molecules/05-navigation/08-ribbon-user-menu/_ribbon-user-menu.scss]: @aCyborg will do for both instances

File: styleguide/source/_patterns/01-molecules/05-navigation/08-ribbon-user-menu/_ribbon-user-menu.scss (hunk 3 above)
Comment (martensc) [inline styleguide/source/_patterns/01-molecules/05-navigation/08-ribbon-user-menu/_ribbon-user-menu.scss]: @aCyborg I tried a viewport check in js but it seemed messy and not super reliable.

**Comment by froboy** [comment]
I chatted with Andy and he's agreed that the Corp-style mobile ribbon behavior was a mistake. I'm removing the box-shadow because it's totally out of context with the rest of the site, but leaving the coloring there for now. If we need we can tweak later, but otherwise this looks good for now. Thanks!

**Pull request merged**